public/stylesheets/styles.css

```diff
@@ -0,0 +1,121 @@
+.btn-spacing {
+  margin: 5px;
+}
+
+.btn-window {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 15px;
+  width: 15px;
+  border-radius: 50%;
+  font-family: 'Roboto', sans-serif;
+}
+
+.btn-close {
+  background-color: #fc5855;
+}
+
+.btn-min {
+  background-color: #fec141;
+}
+
+.btn-open {
+  background-color: #35cb4b;
+}
+
+// .btn-close:hover {
```

Comment:
yeah remove these commented out lines. Also check your editor because I think css requires `/* comments like this */`